includes/discovery/sensors/state/ray.inc.php

```diff
@@ -2,8 +2,18 @@
 
 echo 'RAy Racom State';
 
+
 // System Status (Value : na (0) unknown, ok (1) ok, warning (2) warning, alarm (3) alarm)
 $state = snmp_get($device, "systemStatus.0", "-Ovqe", 'RAY-MIB');
+
+$cmd = gen_snmpget_cmd($device, '.1.3.6.1.4.1.33555.1.1.3.1', '-Oqv', 'RAY-MIB', 'ray');
```

Comment:
why are you not using the built in functions?

if .0 is inconsistent, just use `snmp_getnext($device, "systemStatus", "-Ovqe", 'RAY-MIB')` and it will work for both

Reply:
to find the oid returned, change the query flags:
```
$output = snmp_getnext($device, "systemStatus", "-Oqn", 'RAY-MIB');
if ($output !== false) {
    list($oid, $state) = explode(' ', $output, 2);
    ...
}
```

Reply:
> why are you not using the built in functions?
> 
> if .0 is inconsistent, just use `snmp_getnext($device, "systemStatus", "-Ovqe", 'RAY-MIB')` and it will work for both

If this is not the case, the following OID will be used.